src/clients/java/src/main/java/com/tigerbeetle/AccountFlags.java

```diff
@@ -1,10 +1,27 @@
+//////////////////////////////////////////////////////////
+// This file was auto-generated by java_bindings.zig
+// Do not manually modify.
+//////////////////////////////////////////////////////////
+
 package com.tigerbeetle;
 
 public interface AccountFlags {
-    short NONE = (short) 0;
-    short LINKED = (short) (1 << 0);
-    short DEBITS_MUST_NOT_EXCEED_CREDITS = (short) (1 << 1);
-    short CREDITS_MUST_NOT_EXCEED_DEBITS = (short) (1 << 2);
+    int NONE = (int) 0;
+
+    /**
+     * @see <a href="https://docs.tigerbeetle.com/reference/accounts#flagslinked">linked</a>
+     */
+    int LINKED = (int) (1 << 0);
```

Comment:
Why are these now `int` instead of `short`?

Reply:
For better API ergonomy, we expose 16-bit unsigned integers in Java as `int` instead of `short`.
Even though, they are always validated and stored with the correct type.